packages/rn-tester/js/examples/Modal/ModalExample.js

```diff
@@ -172,10 +172,14 @@ class ModalExample extends React.Component<{...}, $FlowFixMeState> {
             this.setState({currentOrientation: evt.nativeEvent.orientation})
           }
           onDismiss={() => {
-            if (this.state.action === 'onDismiss') alert(this.state.action);
+            if (this.state.action === 'onDismiss') {
```

Comment:
Can we move all the formatting to a separate PR?

Reply:
@lunaleaps I think the linting command added the formatting. I haven't changed anything.

Reply:
Should I manually revert it to the old one ?

Reply:
@lunaleaps formatting reverted 😄